src/modules/package.rs

```diff
@@ -68,6 +68,22 @@ fn get_node_package_version(context: &Context, config: &PackageConfig) -> Option
     Some(formatted_version)
 }
 
+fn get_jsr_package_version(context: &Context, config: &PackageConfig) -> Option<String> {
+    ["jsr.json", "jsr.jsonc", "deno.json", "deno.jsonc"]
+        .iter()
+        .find_map(|file| {
+            let contents = context.read_file_from_pwd(file)?;
```

Comment:
It's likely best to stop after finding the first file, whether it parses correctly or not:
```suggestion
        .find_map(|file| context.read_file_from_pwd(file))
        .and_then(|contents| {
```

Reply:
Done. In quick testing, JSR seems to proiritize `deno.json(c)` files over `jsr.json(c)` files, so I changed it to look for deno config files before JSR config files.